src/nvim/fold.c

```diff
@@ -1742,10 +1742,9 @@ static void foldDelMarker(
       assert(p >= line);
       memcpy(newline, line, (size_t)(p - line));
       STRCPY(newline + (p - line), p + len);
-      ml_replace_buf(buf, lnum, newline, false);
+      ml_replace(lnum, newline, false);
       extmark_splice_cols(buf, (int)lnum-1, (int)(p - line),
-                          (int)len,
-                          0, kExtmarkUndo);
+                          (int)len, 0, kExtmarkUndo);
```

Comment:
Rebase error here too ?

Reply:
hmm what is the problem

Reply:
`ml_replace` should be `ml_replace_buf` sorry for the bad range comment.

Reply:
onto it